xarray/core/dataarray.py

```diff
@@ -1087,6 +1087,29 @@ def isel(
         --------
         Dataset.isel
         DataArray.sel
+
+        Examples
+        --------
+        >>> da = xr.DataArray(
+                np.arange(25).reshape(5, 5),
+                dims=("x", "y")
+            )
```

Comment:
the doctest fails here because we need to also add prompts to the other lines. I usually type / copy the code into a plain `python` shell (no ipython) and then copy the code together with the prompt and the output.
```suggestion
        >>> da = xr.DataArray(
        ...     np.arange(25).reshape(5, 5),
        ...     dims=("x", "y")
        ... )
```